docs/env.md

```diff
@@ -1,117 +1,260 @@
 # Environment variables
 
-We try to follow [12 factors app](https://12factor.net/) and so use
-environment variables for configuration.
+`marsha` contains several projects, two of which are configured through environment variables. There are therefore two `env.d` folders with different environment variables.
 
-Here is a list of the ones that are needed or optional:
+First, there is our Django backend. We try to follow [12 factors app](https://12factor.net/) and so use environment variables for configuration.
 
-## DJANGO_SETTINGS_MODULE
+Then, there is our aws deployment configurations in the `src/aws` folder. It looks in `src/aws/env.d`.
 
-Description
+## 1. Django backend environment
 
-:   Define the settings file to use
+Specified in `{REPO_ROOT}/env.d/development`.
 
-Type
+### General Django settings
 
-:   String
+#### DJANGO_CONFIGURATION
 
-Mandatory
+Define the configuration to use in settings.
 
-:   Yes
+- Type: `String`
+- Required: Yes
+- Default: `Development`
+- Choices: Currently only `Development` is available.
 
-Default
+#### DJANGO_DEBUG
 
-:   None
+Turns on/off debug mode.
 
-Choices
+- Type: Boolean
+- Required: No
+- Default: `True` if `DJANGO_CONFIGURATION` is set to `Development`, `False` otherwise
+- Choices: `True` or `False`
 
-:   Must be set to `marsha.settings`
+#### DJANGO_SECRET_KEY
 
-## DJANGO_CONFIGURATION
+Standard Django secret key used to make the instance unique and generate hashes such as CSRF tokens or auth keys.
```

Comment:
See [Django documentation](https://docs.djangoproject.com/en/dev/ref/settings/#secret-key) for more information about this setting.